src/main/modules/passport/index.ts

```diff
@@ -45,6 +52,13 @@ export class Passport {
 
 }
 
+export function isAuthed(req: Request, res: Response, next: NextFunction) {
```

Comment:
should we make this a middleware before the routes so we dont need to add it to all controllers routes.